utils/material_scan_2D.py

```diff
@@ -10,9 +18,42 @@
 # DD4hep geometry service
 from Configurables import GeoSvc
 
+parser.add_argument(
+    "--compactFile",
+    help="Compact detector file to use",
+    type=str,
+    default=fspath(
+        Path(environ["k4geo_DIR"]) / "ILD" / "compact" / "ILD_sl5_v02" / "ILD_l5_v02.xml"
```

Comment:
This technically changes the default geometry from `IDEA_o1_v02.xml` to an ILD geometry. However, it also looks like the previous version didn't work without editing the file?

Technically this should be a positional (required) argument, since the script can't do anything useful without it. However, at this point `--compactFile` is also pretty much a convention for several scripts.

For me these changes are OK.

Reply:
I don't care which detector is the default as long as the path is absolute or relative to `$K4GEO` or `$k4geo_DIR`. Originally the path was relative and Daniel and I had trouble getting it to work. I think this should work out of the box without assuming a current working dir

Reply:
Originally which path was relative?

Reply:
The path to the `IDEA_o1_v02.xml` compact file.

Reply:
Ahh, ok, the detector file was just hard coded. I guess the compactFile will accept relative or absolute paths.

Reply:
Yes, exactly. And yes, absolute and relative paths will work. It is directly passed to the `GeoSvc.detectors`.